packages/fast-tooling-react/src/navigation/navigation-tree-item.tsx

```diff
@@ -151,6 +117,7 @@ export const DraggableNavigationTreeItem: React.FC<NavigationTreeItemProps> = ({
             dragStart(dictionaryId);
         },
         end(): void {
+            // TODO: investigate why when not dropped on a drop target this takes extra time to respond
```

Comment:
nittiest of nits (worst of the worst comments): Should we have a task for this, or are you already tracking in a branch?

Reply:
No, we should have a task, I forgot I put that in there, fixed!